resources/lang/pl_PL/tables.yml

```diff
@@ -21,6 +21,16 @@ tables:
       createdAt: "$t(models.user.createdAt)"
       facilitiesMember: "$t(models.user.members)"
       hasGlobalAdmin: "$t(models.user.hasGlobalAdmin)"
+      lastLoginFacility:
+        id: "$t(models.user.lastLoginFacility.id)"
+        name: "$t(models.user.lastLoginFacility.name)"
```

Comment:
To się zgadzam że wcale bym nie dawał. Ogólnie jeśli jakiejś kolumny nie chcemy docelowo pokazywać to nie ma co robić tłumaczeń. Wyświetli się ta nazwa klucza i to jest tymczasowo OK.

Reply:
Zrobione.